csi/moac/volume.js

```diff
@@ -23,8 +23,10 @@ class Volume {
   // @params {string[]} spec.requiredNodes   Replicas must be on these nodes.
   // @params {number}   spec.requiredBytes   The volume must have at least this size.
   // @params {number}   spec.limitBytes      The volume should not be bigger than this.
+  // @params {string}   spec.protocol        The share protocol for the nexus.
+  // @params  {object}  status               Current properties of the volume
   //
-  constructor (uuid, registry, spec) {
+  constructor (uuid, registry, spec, status) {
```

Comment:
just for your consideration: if we pass an optional argument size (instead of the whole status), then apis become more clear and we don't leak internal structures from k8s to the rest of the code. If this function is changed then Import volume function can use the size instead of status too.

Reply:
yep that makes sense, will do.